Mediapipe.Net.Examples.OsuFrameworkVisualTests/MediapipeDrawable.cs

```diff
@@ -43,39 +50,87 @@ public MediapipeDrawable()
 
 #pragma warning disable IDE0051
         [BackgroundDependencyLoader]
-        private void load(Camera camera, FrameConverter converter, FaceMeshCpuCalculator calculator)
+        private void load(Camera camera, FrameConverter converter, FaceMeshCpuSolution calculator)
         {
             this.camera = camera;
             this.converter = converter;
             this.calculator = calculator;
-            this.camera.OnFrame += onFrameEventHandler;
-            this.camera.StartCapture();
+
+            frameLoopThread = new Thread(frameLoop);
```

Comment:
do you think it's safe to spawn our own threads in this case?

Reply:
The MediaPipe code is safe as long as it runs on one specific thread. The only problem right now with the osu!framework example is that it can't be exited cleanly because this thread keeps running unless you press CTRL+C in the terminal.
I've been trying to make it multithreading-friendly since right now this setup means that the time the calculator takes to process one image is the bottleneck of FPS, but so far even with my Solution outputs object it doesn't seem like I've got a solution...